Dragonflight/WarriorArms.lua

```diff
@@ -1427,10 +1501,16 @@ spec:RegisterAbilities( {
         range = 8,
 
         handler = function ()
-            if talent.in_for_the_kill.enabled then applyBuff( "in_for_the_kill" ) end
+            if talent.in_for_the_kill.enabled then 
+                applyBuff( "in_for_the_kill" ) 
+                stat.haste = stat.haste + ( target.health.pct < 35 and 0.2 or 0.1 )
```

Comment:
Same note re: only modifying the haste if In for the Kill wasn't already applied.

Reply:
CS fixed in https://github.com/Hekili/hekili/pull/1670/commits/33370bc31a64a614f1d30da0c18027f72e0beb2f
WB fixed in https://github.com/Hekili/hekili/pull/1670/commits/4cc68109cde878a259f4b46c5c1ad90194cca215